RocketChat/Rocket.Chat.ReactNative
# [NEW] Teams Directory

## Proposed changes


Adds Teams to the `DirectoryView`

## Issue(s)	


## How to test or reproduce


1. Go to Directory
2. Tap on the `DropDown` item which says `Channels`
3. Select Teams

A directory of teams should render

## Screenshots

<table>
  <thead>
    <tr>
      <th>Before</th>
      <th>After (those are channels that were converted to teams)</th>
      </thead>
    </tr>
    <tbody>
      <tr>
        <td>
        <img width="350" alt="image" src="https://user-images.githubusercontent.com/36211892/120510059-0f5a8280-c397-11eb-96b8-e67e063550f1.png">
        </td>
        <td>
        <img width="350" alt="image" src="https://user-images.githubusercontent.com/36211892/120510176-2ac58d80-c397-11eb-92ee-8963efde7919.png">
        </td>
      </tr>
    </tbody>
</table >

## Types of changes




## Checklist



## Further comments
